lib/node_modules/@stdlib/_tools/scaffold/math-strided-real-typed-unary/scripts/runner.js

```diff
@@ -0,0 +1,279 @@
+function keywordsEnvVar( keywords ) {
+	return 'KEYWORDS=\'' + keywords.join( ',' ) + '\'';
+}
+
+/**
+* Returns a `C_X_VALUES_LEN_8` environment variable string.
+*
+* @private
+* @param {NumberArray} values - example values
+* @param {string} dtype - values data type
+* @returns {string} environment variable string
+*
+* @example
+* var values = [ 1, 2, 3, 4, 5, 6, 7, 8 ];
+*
+* var s = valuesEnvVar( values, 'int32' );
+* // returns 'C_X_VALUES_LEN_8=\'1,2,3,4,5,6,7,8\''
+*/
+function valuesEnvVar( values, dtype ) {
+	var vals;
+	var v;
+	var i;
+
+	if ( isRealFloatingDataType( dtype ) ) {
+		vals = [];
+		for ( i = 0; i < 8; i++ ) {
+			v = values[ i ];
+			vals.push( v.toString() );
+			if ( isInteger( v ) ) {
+				vals[ i ] += '.0';
+			}
+		}
+	} else {
+		vals = values.slice( 0, 8 );
+	}
+	return 'C_X_VALUES_LEN_8=\'' + vals.join( ',' ) + '\'';
+}
+
+/**
+* Returns a string containing environment variable for generating pseudorandom numbers.
+*
+* @private
+* @param {Object} obj - configuration object
+* @returns {string} environment variable string
+*
+* @example
+* var o = {
+*     'prng': 'random/base/uniform',
+*     'parameters': [ -100, 100 ]
+* };
+*
+* var s = prngEnvVars( o );
+* // returns 'BASE_PRNG=\'uniform\',RAND_MIN=-100,RAND_MAX=100'
+*/
+function prngEnvVars( obj ) {
+	var base;
+	var min;
+	var max;
+	var p;
+
+	p = obj.prng.split( '/' );
+	base = p[ p.length-1 ];
+	min = obj.parameters[ 0 ];
+	max = obj.parameters[ 1 ];
+	if ( base === 'uniform' ) {
+		if ( isInteger( min ) ) {
+			min = min.toString() + '.0';
+		}
+		if ( isInteger( max ) ) {
+			max = max.toString() + '.0';
+		}
+	} else {
+		min = min.toString();
+		max = max.toString();
+	}
+	return [
+		'BASE_PRNG=\'' + base + '\'',
+		'C_RAND_MIN=' + min,
+		'C_RAND_MAX=' + max
+	].join( ' ' );
+}
+
+/**
+* Returns a `JAVASCRIPT_RETURN_TYPE` environment variable string.
+*
+* @private
+* @param {string} dt - return type
+* @returns {string} environment variable string
+*
+* @example
+* var s = returnTypeEnvVar( 'number' );
+* // returns 'JAVASCRIPT_RETURN_TYPE=\'number\''
+*/
+function returnTypeEnvVar( dt ) {
+	return 'JAVASCRIPT_RETURN_TYPE=\'' + dt + '\'';
+}
+
+
+// MAIN //
+
+/**
+* Main execution sequence.
+*
+* @private
+*/
+function main() {
+	var keys;
+	var envs;
+	var cmd;
+	var p;
+	var k;
+	var v;
+	var o;
+	var i;
+	var j;
+
+	for ( i = 0; i < DATA.length; i++ ) {
+		o = DATA[ i ];
+		p = resolve( ROOT_DIR, '@stdlib/math/strided/special', o.alias, 'package.json' );
+		if ( existsSync( p ) ) {
+			if ( CREATE_ONLY ) {
+				log( 'Package already exists. Skipping @%s...', 'stdlib/math/strided/special/' + o.alias );
+				continue;
+			}
+			log( 'Updating package: @%s...', 'stdlib/math/strided/special/' + o.alias );
+		} else {
+			log( 'Creating package: @%s...', 'stdlib/math/strided/special/' + o.alias );
+		}
+		keys = objectKeys( o );
+		envs = [];
+		for ( j = 0; j < keys.length; j++ ) {
+			k = keys[ j ];
+			v = o[ k ];
+			if ( k === 'alias' ) {
```

Comment:
You need to use the "base_alias", not the alias. E.g., `absf` needs to be map to `sabs`.